diff --git a/evmd/cmd/evmd/config/config.go b/evmd/cmd/evmd/config/config.go
index 65f30747..2699ca0b 100644
--- a/evmd/cmd/evmd/config/config.go
+++ b/evmd/cmd/evmd/config/config.go
@@ -31,13 +31,6 @@ var ChainsCoinInfo = map[uint64]evmtypes.EvmCoinInfo{
 		DisplayDenom:  "test",
 		Decimals:      evmtypes.EighteenDecimals,
 	},
-	// SixDecimalsChainID provides a chain ID which is being set up with 6 decimals
-	SixDecimalsChainID: {
-		Denom:         "utest",
-		ExtendedDenom: "atest",
-		DisplayDenom:  "test",
-		Decimals:      evmtypes.SixDecimals,
-	},
 }
 
 const (
